kubernetes: Fix examples for kubernetes 0.14.0

stefwalter opened the pull request:
And bump our development version. Incompatible changes were made
to the service ports specification, among other things.

stefwalter commented:
@petervo This provides valid examples that work with our little cluster.

petervo commented:
Just wondering why we removed namespace from some of them, it doesn't break the manifest. Was it just to change what we are testing?

stefwalter commented:
> Just wondering why we removed namespace from some of them, it doesn't break the manifest. Was it just to change what we are testing?

Because it doesn't work with the 'Deploy Application' code, which expects to put the namespace in later. We should probably make that less fragile, but for now I removed it from the examples.
